modules/ztp-add-local-reg-for-sno-duprofile.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: CONCEPT
+[id="ztp-add-local-reg-for-sno-duprofile_{context}"]
+= Configuring the Image Registry Operator for local caching of images
+
+{Product} manages image caching with a local registry. A Single Node OpenShift (SNO) Distributed Unit (DU) is subject to bandwidth restrictions when communicating with the centralized registry. This can lead to long image download times.
+
+This is unavoidable for the initial deployment. Over time, there is a risk that `crio` will wipe `/var/lib/containers/storage` in the case of a disorderly shutdown.
+
+To address this, you need a local registry on the SNO. The cluster images need to be  mirrored to this registry along with being used as the registry for the applications.
```

Comment:
I think it's important to clarify that the image registry can only be used for user application images and cannot be used for the OpenShift release or OLM operators images, see https://issues.redhat.com/browse/RFE-2488

Reply:
I have addressed this comment in the current preview.